Makefile

```diff
@@ -29,7 +29,11 @@ install-dev-tools:  ## Installs all necessary cargo helpers
 	cargo install cargo-nextest --locked
 	cargo install cargo-binstall
 	cargo binstall cargo-risczero
+ifeq ($(shell uname -mo), Darwin x86_64)
```

Comment:
indent right please

Reply:
> indent right please

https://stackoverflow.com/questions/4483313/make-error-for-ifeq-syntax-error-near-unexpected-token